tests/sb-util.test.ts

```diff
@@ -89,11 +89,13 @@ describe('ScratchProject class --------------------', () => {
 			test('to query sprites where value is a string with double quotes', () => {
 				const sprite = sp.sprites('[name="Sprite1"]');;
 				expect(sprite).toBeInstanceOf(SpriteCollection);
+				expect(sprites.first()).toBeInstanceOf(Sprite);
```

Comment:
This doesn't match the test description. Probably needs its own test wrapper.

Reply:
Good catch! Thanks. I think this was a debugging line.